frontend/public/models/yaml-templates.ts

```diff
@@ -338,6 +354,17 @@ apiVersion: apps.openshift.io/v1
 kind: DeploymentConfig
 metadata:
   name: example
+  annotations:
+    image.openshift.io/triggers: |-
```

Comment:
DeploymentConfigs have first class API support for triggers and don't need to use the annotation.

```
$ oc explain deploymentconfig.spec.triggers
KIND:     DeploymentConfig
VERSION:  apps.openshift.io/v1

RESOURCE: triggers <[]Object>

DESCRIPTION:
     Triggers determine how updates to a DeploymentConfig result in new
     deployments. If no triggers are defined, a new deployment can only occur as
     a result of an explicit client update to the DeploymentConfig with a new
     LatestVersion. If null, defaults to having a config change trigger.

     DeploymentTriggerPolicy describes a policy for a single trigger that
     results in a new deployment.

FIELDS:
   imageChangeParams    <Object>
     ImageChangeParams represents the parameters for the ImageChange trigger.

   type <string>
     Type of the trigger
```

Reply:
@spadgett Removed the trigger for Pod and DeploymentConfigs.
Do we need to revert the image to `openshift/hello-openshift` for Pod and DeploymentConfigs?

Reply:
@cyril-ui-developer You shouldn't remove the trigger for DeploymentConfigs. Instead update it to use the `spec.triggers` instead of the annotation.

I'd just use `registry.redhat.io/openshift4/ose-hello-openshift-rhel8` directly for pods.

Reply:
@spadgett Updated, thanks! However the status is still `ImagePullBackOff` instead of `Running`. 

![Screen Shot 2020-11-16 at 2 14 34 PM](https://user-images.githubusercontent.com/15249132/99297400-36e06800-2816-11eb-8e14-aa94d0dd2cb8.png)
![Screen Shot 2020-11-16 at 2 14 52 PM](https://user-images.githubusercontent.com/15249132/99297402-3778fe80-2816-11eb-9e0b-852f710adcd7.png)